qiskit_experiments/database_service/db_analysis_result.py

```diff
@@ -417,6 +422,30 @@ def auto_save(self, save_val: bool) -> None:
             self.save()
         self._auto_save = save_val
 
+    @staticmethod
+    def _format_db_display(value):
+        if isinstance(value, (int, bool)):
+            return value
+        if isinstance(value, float):
+            if math.isfinite(value):
+                return value
+            else:
+                return serialize_safe_float(value)["__value__"]
+
+        # Values that we can store by string conversion
+        if isinstance(value, complex):
+            # Convert to string and truncate to 6 significant digits
+            value = "{:.8g}".format(value)
+        elif isinstance(value, (list, np.ndarray)) and len(value) < 10:
+            value = np.array2string(
+                np.asarray(value), separator=", ", max_line_width=100, precision=8
+            )
+
+        # Store string
+        if isinstance(value, str) and len(value) < 70:
+            return value
```

Comment:
Why not truncate the length at 70 instead of not saving at all if it's over?

Reply:
Do you think there is any point in displaying part of a result?

Reply:
Depends on when this might happen...if eventually we're storing some long string that's part of the results for some reason, I assume it would be valuable to preview it? Maybe it would be some kind of useful diagnostics message passed from hardware, or some additional environmental data associated with the experiment?